src/app/Providers/components/ProvidersTable/OpenShift/OpenShiftProvidersTable.tsx

```diff
@@ -168,7 +200,7 @@ const OpenShiftProvidersTable: React.FunctionComponent<IOpenShiftProvidersTableP
 
   return (
     <>
-      <Level>
+      <Level className="pf-u-pb-sm">
```

Comment:
Not that it matters much but I think we can import this class from `'@patternfly/react-styles/css/utilities/Spacing/spacing'` like we do elsewhere